### PR TITLE
Supermatter on Bagel!! (and it's warppoint)

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

the title ^^ !!

## Why / Balance

let's transform a 0 point 1 by 1 square meter TEG room into a functional Supermatter to TEG Engine. Firstly, remove the burn chamber and use galvanized square steel to make two entrances that extend outward. I borrow some screws from my aunt to secure the radiation collectors. Cover the frames with neat clean steel pipes. Use leftover pipes as...

## Media

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl:
- add: Bagel: Supermatter Engine

### CODE REVIEW

**Comment by a-person5660** [comment]
next do a map on the high pop pool

**Comment by whateverusername0** [comment]
> next do a map on the high pop pool

idk what maps are high pop, i've done fland already doe

**Comment by a-person5660** [comment]
> idk what maps are high pop, i've done fland already doe

fland, oasis and idr the other one lol

**Comment by Piras314** [comment]
Bagel is the other highpop map you're thinking of I'm pretty sure lol. But yeah we need SM on Oasis next.

**Comment by whateverusername0** [comment]
> Bagel is the other highpop map you're thinking of I'm pretty sure lol. But yeah we need SM on Oasis next.

wait for this to get merged 😭

File: Content.Server/Goobstation/Supermatter/Systems/SupermatterSystem.cs
Comment (jorgun) [inline Content.Server/Goobstation/Supermatter/Systems/SupermatterSystem.cs:1]: why is this in a map update?

Comment (whateverusername0) [reply Content.Server/Goobstation/Supermatter/Systems/SupermatterSystem.cs:1]: i just fixed formatting a bit, nothing really changed (except sm now has a warppoint but i stated it in teh title, durk asked for it)

**Review by Aidenkrz — APPROVED** [review]

**Pull request merged**